sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_operations/_patch.py

```diff
@@ -0,0 +1,226 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+
+import time
+from typing import Any, IO, Callable, List, Optional, Union, cast
+
+from azure.core.polling import PollingMethod, LROPoller, NoPolling
+
+from azure.confidentialledger._operations._operations import (
+    ConfidentialLedgerClientOperationsMixin as GeneratedOperationsMixin,
+)
+from azure.confidentialledger._operations._operations import JSON
+
+__all__: List[str] = [
+    "ConfidentialLedgerClientOperationsMixin"
+]  # Add all objects you want publicly available to users at this package level
+
+
+def patch_sdk():
+    """Do not remove from this file.
+
+    `patch_sdk` is a last resort escape hatch that allows you to do customizations
+    you can't accomplish using the techniques described in
+    https://aka.ms/azsdk/python/dpcodegen/python/customize
+    """
+
+
+class StatePollingMethod(PollingMethod):
+    """Polling method for methods returning responses containing a 'state' field; the polling
+    completes when 'state' becomes a desired value.
+    """
+
+    def __init__(
+        self,
+        operation: Callable[[], JSON],
+        desired_state: str,
+        polling_interval_s: float,
```

Comment:
`polling_interval_s` is weird, maybe just `polling_interval`?

Reply:
My preference is to explicitly include the units - I can change it if you really want me to 😄

Reply:
ohhhh was this meaning seconds? It looks like you're trying to do `polling_intervals`. It looks kinda weird but as long as its not public it doesn't matter